recipes/xorg/all/conanfile.py

```diff
@@ -54,7 +54,7 @@ def system_requirements(self):
             elif tools.os_info.with_yum or tools.os_info.with_dnf:
                 packages = ["libxcb-devel", "libfontenc-devel", "libXaw-devel", "libXcomposite-devel",
                             "libXcursor-devel", "libXdmcp-devel", "libXft-devel", "libXtst-devel", "libXinerama-devel",
-                            "xorg-x11-xkb-utils-devel", "libXrandr-devel", "libXres-devel", "libXScrnSaver-devel", "libXvMC-devel",
+                            "libxkbfile-devel", "libXrandr-devel", "libXres-devel", "libXScrnSaver-devel", "libXvMC-devel",
```

Comment:
I think we should make a switch based on the OS version in order to support old and new versions

Reply:
it should be fine: libxkbfile-devel is a requirement of xorg-x11-xkb-utils-devel, which provides a single file: xkbcomp.pc, and xkbcomp is not referenced in https://github.com/conan-io/conan-center-index/blob/master/recipes/xorg/all/conanfile.py#L81. Let's wait for the next run on https://github.com/bincrafters/system-packages-checks/actions/workflows/cron.yml

Reply:
the only failure is on fedora 34, but it is unrelated: conan settings do not have gcc 11 yet : https://github.com/bincrafters/system-packages-checks/runs/2482530226?check_suite_focus=true#step:6:108

Reply:
Thanks for the information!